### DISCUSSION TITLE
How do I add a scrollbar to a Label?

### DISCUSSION THREAD

**Discussion by totordev** [Textualize/textual#2403] [Q&A] [answered]
Is it possible to add a scrollbar to a Label object? If yes, how?

**Comment by willmcgugan** [comment, marked as answer]
Kind of. But you would add the label as a child of a scrollable container. If your label is larger than the container you will get a scrollbar.

```python
with VerticalScroll():
    yield Label(LOTS_OF_TEXT)
```

**Reply by totordev** [reply]
When I try to do this, it tells me there is a syntax error

**Reply by willmcgugan** [reply]
Are you on the Textualize Discord? It might be a better place to ask Python questions.

**Reply by totordev** [reply]
No, I am not, yet. I had thought of joining it.

**Reply by totordev** [reply]
Is there a link I can use to join the server? I can't find it in the Guild Discovery tab.

**Reply by totordev** [reply]
Oh, nevermind I found the link.
